### ISSUE TITLE
Unable to detect combine % full

### ISSUE THREAD

**Issue by Psytronex** [netjungle/courseplay#47]
After creating a route for the Tractor and manually selecting a combine harvester, the tractor waits with the status "Waiting until fill level is reached".  

Fill levels on the tractor are: start at %: 50; drive on at %: 90.  Now, regardless of what percentages you set these at, and regardless of how empty or full the harvester is, and with a worker hired for the harvester, the tractor is never triggered to go to the harvester.  This includes if the harvest is below 50% and then fills to above 50%.

In the courseplay settings for the harvester, the name of the tractor appears.  The name of the harvest appears in the tractor settings, so they do know about each other.

Type: Combi (empty combine and offload).  Version: 3.2

**Comment by Psytronex** [comment]
Exiting the game and restarting it seems to have allowed the harvester and tractor to communicate properly.

**Comment by JakobTischler** [comment]
Was that a combine or a harvester (potatoes/beets)? Also, have you checked if realistic driving was activated? If so, that could have been the cause.

**Comment by Psytronex** [comment]
It was a combine (as in wheat).

Where is this "realistic driving" option of which you speak?

**Comment by JakobTischler** [comment]
Sorry I used the wrong term. It's the "pathfinding" option. If activated, the driver will not drive to the combine if the fruit is on the combine's left hand side - even when the combine's fill level is above the set percentage.

**Comment by Psytronex** [comment]
The pathfinding option was on, on both the tractor and the combi (although the combi wasn't driving a route).  When I restarted the game, which fixed the problem, the settings remained the same.  The only difference was the restart.

I was harvesting wheat, so I'm not sure if your "fruit" comments apply to my situation.  

I've tested the tractor, now, with the pathfinding option on and off, and the tractor is able to talk to the combine successfully.  So I don't see the same behaviour.  As far as I can see, it was an anomaly fixed by restarting the game.
